components/Layout/Header.tsx

```diff
@@ -0,0 +1,54 @@
+import Logo from "../assets/images/logo/logo.svg";
+import React from "react";
+import { useRouter } from "next/router";
+import Image from "next/image";
+import Link from "next/link";
+import styles from "./Header.module.css";
+
+const Header: React.FC = () => {
+  const router = useRouter();
+  const { pathname } = router;
+
+  const getLinkStyle = (isActive: boolean) => ({
+    color: isActive ? "var(--blue)" : undefined,
+  });
```

Comment:
P3:
해당 함수가 실행되면 `CSSProperties`를 반환하니 아래처럼 반환값을 명시해주시는 것이 더 좋을 것 같습니다.

```suggestion
const getLinkStyle = (isActive: boolean): CSSProperties['color'] => {
  return { color: isActive ? "var(--blue)" : undefined };
}
```